example/src/main/java/com/spruceid/mobilesdkexample/wallet/GenericCredentialListItem.kt

```diff
@@ -159,18 +160,14 @@ fun GenericCredentialDetailsItem(credentialPack: CredentialPack) {
                     "credentialSubject.driversLicense.family_name",
                     "credentialSubject.driversLicense.birth_date",
                 ),
-                formatter = {values ->
-                    val w3cvc = values.toList()
-                        .first { (key, _) ->
-                            val credential = credentialPack.getCredentialById(key)
-                            (credential?.asJwtVc() != null || credential?.asJsonVc() != null)
-                        }.second
+                formatter = { values ->
+                    val w3cvc = values.toList().first().second
```

Comment:
@Juliano1612 can I revert this change? It's also possible for the value to be an mDoc, in which case `w3cvc["credentialSubject"]` will always be null

Reply:
@cobward, you can revert, but @w4ll3 and I are completely refactoring this component. I updated it to keep the app working, but I wouldn't worry about that now.